### ISSUE TITLE
Install on other drives/Custom filepath

### ISSUE THREAD

**Issue by Stev-eH** [bboettcher3/gRainbow#92]
Currently only able to unpack/install on the C-Drive on windows, please add the ability to install to any filepath

**Comment by CLFabula** [comment]
Yeah, or maybe just tell us where the installation is located... I installed it and now i have no clue where the vst is, i have to put the vst into my vst folder and i cant find it on my drive.

**Comment by bboettcher3** [comment]
> Yeah, or maybe just tell us where the installation is located... I installed it and now i have no clue where the vst is, i have to put the vst into my vst folder and i cant find it on my drive.

As for most plugins, **C:\Program Files\Common Files\VST3**
